File: Javascript Solutions/Ejercicio3.js
var array = [parseInt(Math.random() * 11),parseInt(Math.random() * 11),parseInt(Math.random() * 11),parseInt(Math.random() * 11),parseInt(Math.random() * 11)];

console.log("Entrada: "+array)
function sumarElementos(elementos){
    for (var i=0;i<elementos.length;i++){
        if (i>0){
            elementos[i] += elementos[(i-1)];
        }
    }
    console.log("Salida: "+elementos);
}

sumarElementos(array);

File: Python Solutions/Ejercicio3.py
import random
array = [int(random.random()*11),int(random.random()*11),int(random.random()*11),int(random.random()*11),int(random.random()*11)]
print("Entrada: "+str(array))
i = 0
def sumarElementos(elementos):
    for i in range(1,len(elementos)):
        if i>0:
            elementos[i]+=elementos[(i-1)]
    print("Salida: "+str(elementos))

sumarElementos(array)